### ISSUE TITLE
Resources loaded upon adding module to position

### ISSUE THREAD

**Issue by pau1a** [gantry/gantry5#2501]
Hi guys,

I doubt this is an issue, probably more of a question but maybe a flaw in docs.

Im working on adding a custom html module to hydrogen.

The module requires loading a specific version of jQuery however I suspect this is being compromised by the template externally loading the following resources:

	<script src="/media/jui/js/jquery.min.js?f203add6aadf0964f467ac96c5865869"></script>
	<script src="/media/jui/js/jquery-noconflict.js?f203add6aadf0964f467ac96c5865869"></script>
	<script src="/media/jui/js/jquery-migrate.min.js?f203add6aadf0964f467ac96c5865869"></script>
	<script src="/media/jui/js/bootstrap.min.js?f203add6aadf0964f467ac96c5865869"></script>

My tests suggest these resources (and others) are automatically added when any module is added to a position, be it a "module position" or a "module instance".

So in a nutshell, is this behaviour a part of anticipated functionality? Can it be disabled in order for me to manage the version of jQuery added if even for certain outlines and/or have I made a complete howler and missed something ridiculously obvious?

**Comment by mahagr** [comment]
These resources are automatically added in Joomla as unfortunately most of the modules do not add them even if they need those. This is because of the default templates in Joomla always load all of those javascript files and everyone assumes that those are loaded.

**Comment by pau1a** [comment]
Hi Matthias, Thanks for that, thats interesting. So does this mean that in reality the Atoms which load these resources are really just taking care of this in the (unlikely) event that the site has no module positions loaded in the current outline?

Would you be prepared to share where in the gantry source this takes place as Id like to try a bit of tinkering?

**Comment by mahagr** [comment]
Yes, there's a way to load them also manually by atoms if needed. But as we were getting too many support requests, we had to force it on. 

BTW, also Joomla module editor (and content) needs these turned on, so if you log in, it also forces jQuery to be on.

There is really no way to turn off this feature, the code is in:
https://github.com/gantry/gantry5/blob/develop/src/platforms/joomla/classes/Gantry/Framework/Platform.php#L260
